src/FourierTransforms.jl/ctfft.jl

```diff
@@ -45,7 +54,7 @@ plan_inv(p::CTPlan{T}) where T =
     ScaledPlan(invCT(p), normalization(real(T), p.n, 1))
 
 # steps for pregenerated kernels:
-function ωpow(T::Type{<:Complex}, n, i)
+function ωpow(T::Type{<:Complex{<:AbstractFloatOrInterval}}, n, i)
```

Comment:
Widen to `Real` instead

Reply:
I did that in the first place, but I do not think that's the right thing to do here.

There is some discussion in IntervalArithmetic.jl to make (some) intervals a subtype of AbstractFloat. That seems more logical, because intervals satisfy some promises that Reals might not (e.g. they implement `eps`) and are in that sense floating point numbers.

To have a counterexample: Note, that `Int <: Real`, but the FFT code is certainly not suited for  `Int`.

Reply:
Sure but in the meantime it's easier as a hack to widen the FFT code more than necessary than to add a dep on intervals

Reply:
If it's the only place we truely need the intervals in the main code, then I agree. I only changed it after I noticed the midpoint thing below.